app/views/coronavirus_local_restrictions/_tier_one_restrictions.html.erb

```diff
@@ -1,19 +1,23 @@
 <%
+  tier_heading_pretext = t("coronavirus_local_restrictions.results.level_one.heading_pretext")
+  tier_heading_label = t("coronavirus_local_restrictions.results.level_one.heading_tier_label")
+  tier_heading = "#{tier_heading_pretext} <span class=\"coronavirus-local-restriction__heading-tier-label\">#{tier_heading_label}</span>"
+
   title = if @restriction.present? && @restriction.future.present?
     t("coronavirus_local_restrictions.results.changing_levels_title")
   else
-    t("coronavirus_local_restrictions.results.level_one.heading")
+    tier_heading
   end
 %>
 
-<% content_for :title, title %>
+<% content_for :title, strip_tags(title) %>
```

Comment:
I'm a bit confused about why `strip_tags` is needed here and not on line 29. Surely the natural wrapping needs to be allowed in both places? Is line 29 a fluke because the text for `tier_heading` happens to fit on one line?

Reply:
this is for the browser title so any HTML will display as text